src/why-rust.md

```diff
@@ -5,3 +5,20 @@ Some unique selling points of Rust:
 * Compile time memory safety.
 * Lack of undefined runtime behavior.
 * Modern language features.
+
+<details>
+
+Make sure to ask the class which languages they have experience with.
+Depending on the answer you can highlight different features of Rust:
+
+* Experience with C or C++: Rust eliminates a whole class of _runtime
+  errors_ via the borrow checker. You get performance like in C and
+  C++, but you don't have the memory unsafety issues. In addition, you
+  get a modern language with constructs like pattern matching and
+  built-in dependency management.
+
+* Experience with Java, Python, JavaSript...: You get the same memory
+  safety as in those languages, plus a similar high-level language
+  feeling. In addition you get performance like in C and C++.
```

Comment:
Not just performance but predictability (no surprise pauses for the GC), and potentially lower-level access to hardware.